install_helm.sh

```diff
@@ -0,0 +1,331 @@
+#!/usr/bin/env bash
+
+# Copyright The Helm Authors.
```

Comment:
Lets put the [from-script](https://helm.sh/docs/intro/install/#from-script) snippet inside the `install_helm.sh`, rather then the whole install script.

https://helm.sh/docs/intro/install/#from-script

Reply:
> Lets put the [from-script](https://helm.sh/docs/intro/install/#from-script) snippet inside the `install_helm.sh`, rather then the whole install script.
> 
> https://helm.sh/docs/intro/install/#from-script

I did that initially, but thought why should test script be downloading the helm install stcript every time.

Reply:
> > Lets put the [from-script](https://helm.sh/docs/intro/install/#from-script) snippet inside the `install_helm.sh`, rather then the whole install script.
> > https://helm.sh/docs/intro/install/#from-script
> 
> I did that initially, but thought why should test script be downloading the helm install script every time.

Also, if the download operation fails for any reason, the test would fail too. So I thought having the whole install script would be better for fast installation.

Reply:
> > I did that initially, but thought why should test script be downloading the helm install script every time.
> 
> I dont think that really matters since from looking into the install script its downloading the helm binary anyway.
> 
> > Also, if the download operation fails for any reason, the test would fail too. So I thought having the whole install script would be better for fast installation.
> 
> as mentioned above I dont think its really an issue since the script is downloading the binary anyway. Not mentioning that we would need to then maintain that script, which is handling all sorts of arch/os which is really irrelevant for us.
> 
> I would just put here https://helm.sh/docs/intro/install/#from-script snippet

Then, we'll have to update the `${HELM_INSTALL_DIR:="/usr/local/bin"}` to `${HELM_INSTALL_DIR:="/tmp"}`  in the downloaded get_helm.sh file before the installation start, so that it does not require root permission.